src/mixins/atoms.js

```diff
@@ -1,4 +1,5 @@
 import * as THREE from "three";
+import {CSS2DObject} from "three-css2drender"
```

Comment:
Same as https://github.com/Exabyte-io/wave.js/pull/38/files#r331605278 - why not use rendered from `THREE`?